src/LabLog/Controllers/AdminController.cs

```diff
@@ -54,8 +51,8 @@ public IActionResult RebuildReadModel()
                     .OrderBy(o => (o.Version));
                 _school.ReplaySchoolEvents(schoolEvents);

```

Comment:
I can't see where schoolEvents is defined. Is it class level? Should it start with an underscore?

Reply:
It is defined just before:
```
var schoolEvents = _db.LabEvents
                    .Where(w => (w.SchoolId == e.SchoolId))
                    .OrderBy(o => (o.Version));
_school.ReplaySchoolEvents(schoolEvents);
```
It is a local variable, but not a protected one. What is the convention regarding underscores in variable names?

Reply:
That's what I get for reviewing a PR on my phone :P